.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -60,57 +59,53 @@ body:
 
         Please include screenshots (you can drag any image here) where applicable. Redact any sensitive information.
     validations:
-      required: true
+      required: false
   - type: textarea
     attributes:
       label: "System information and logs"
       description: |
-        Provide details about the environment you experienced the issue in. Change msentraid for the provider you are using:
-      value: |
-        ### Environment
-        * broker version: please run `snap info authd-msentraid`
-        * authd version: please run `/usr/libexec/authd version`
-        * gnome shell version: please run `apt policy gnome-shell`
-        * Distribution: (**NAME** in `/etc/os-release`)
-        * Distribution version: (**VERSION_ID** on `/etc/os-release`):
-
-        ### Log files
-        Please redact/remove sensitive information:
-
-        #### Authd entries:
-
-        ```raw
-        journalctl -u authd.service
-        ```
-
-        ####  MS Entra ID broker entries:
-
-        ```raw
-        journalctl -u snap.authd-msentraid.authd-msentraid.service
-        ```
-
-        ### Application settings
-
-        Please redact/remove sensitive information:
-
-        ####  Broker configuration:
-
-        ```raw
-        cat /var/snap/authd-msentraid/current/broker.conf
+        Copy this command and run it in a terminal. It will open a text editor with the system information and logs. Paste the output below. Redact any sensitive information from the logs.
+        ```bash
+        gnome-text-editor --standalone - <<EOF
```

Comment:
I would use something that is compatible with `$EDITOR` (or `$VISUAL_EDITOR`), as server users may not have any UI.

Reply:
Good point. Not all editors support the same arguments to read input from stdin (`echo foo | vim -` works and so does `echo foo | nano -`, but `echo foo | emacs -` doesn't work), so maybe we should just use `$PAGER` when `gnome-text-editor` is not available.

Reply:
Or we just print it to stdout if `gnome-text-editor` is not available, because I expect that for many users it's actually easier to copy a long text from their terminal than from `less`.

Reply:
The version I just pushed now uses a temporary file with `xdg-open` or `$EDITOR`. That way we should use the editor the user has configured as their default. Another advantage of using the temporary file is that the editor won't ask to save the file before closing.